Makefile

```diff
@@ -223,7 +223,6 @@ build-static: ## Build the static binaries.
 	mkdir -p bin
 	cp -r result/bin bin/static
 
-.PHONY: crio.conf
```

Comment:
@bitoku does `make` detects the state of the `crio.conf` file correctly? I haven't checked myself, admittedly.

Reply:
I will answer myself: it depends on `bin/crio` status, which is probably good enough.